pkg/sql/plan/build_ddl.go

```diff
@@ -66,6 +66,11 @@ func buildCreateTable(stmt *tree.CreateTable, ctx CompilerContext) (*Plan, error
 			})
 		// todo confirm: option data store like this?
 		case *tree.TableOptionComment:
+			if getNumOfCharacters(opt.Comment) > maxLengthOfTableComment {
```

Comment:
why not use engine.CommentDef?

Reply:
engine.CommentDef is used for table comment, not a comment for a column